src/package.xml

```diff
@@ -2447,6 +2486,7 @@
         <members>General_Accounting_Unit__c.All</members>
         <members>Level__c.All</members>
         <members>Opportunity.To_Be_Acknowledged</members>
+        <members>Tribute__c.All</members>
```

Comment:
I don't think there should be a list view included for the Tribute object. There is no packaged tab for this.